File: Practice_C++_Programing/MaxMin.cpp
#include <iostream>

using namespace std;

int FindMax(int[], int);
int FindMin(int[], int);

int main(int argc, char const *argv[])
{
    int arr[5] = {10, 3, 5, 22, 7};
    int max = 0, min = 0;

    max = FindMax(arr, 5);
    min = FindMin(arr, 5);

    cout << "Max = " << max << endl;
    cout << "Min = " << min << endl;

    return 0;
}

int FindMax(int arr[], int length)
{
    int temp = arr[0], max = 0;

    for (int i = 1; i < 5; i++)
    {
        if (temp < arr[i])
        {
            temp = arr[i];
        }
    }

    max = temp;

    return max;
}

int FindMin(int arr[], int length)
{
    int temp = arr[0], min = 0;

    for (int i = 1; i < length; i++)
    {
        if (temp > arr[i])
        {
            temp = arr[i];
        }
    }

    min = temp;

    return min;
}

File: Practice_Python_Programing/MaxMin.py
def find_max(list):
    temp = list[0]

    for i in range(len(list)):
        if temp < list[i]:
            temp = list[i]

    return temp

def find_min(list):
    temp = list[0]

    for i in range(len(list)):
        if temp > list[i]:
            temp = list[i]
    
    return temp

a = []
n = int(input("Enter the number of elements: "))

for i in range(n):
    element = input(f"Enter element {i+1}: ")
    a.append(element)

print("List: ", a)

max = find_max(a)
min = find_min(a)

print("Max: ", max)
print("Min: ", min)